setup.py

```diff
@@ -89,11 +89,13 @@
     'invenio-workflows-files~=0.0,>=0.0.6',
     'invenio-workflows-ui~=1.0,>=1.1.1',
     'invenio-workflows~=6.0,>=6.1.0',
+    'isbn_hyphenate>=1.0.4',
     'langdetect~=1.0,>=1.0.7',
     'librabbitmq~=1.0,>=1.6.1',
     'nameparser~=0.0,>=0.5.3',
     'orcid~=0.0,>=0.7.0',
     'plotextractor~=0.0,>=0.1.6',
+    'pybtex>=0.21',
```

Comment:
We are using a specific notation to denote dependencies. This should be `pybtext~=0.0,>=0.21`.